src/main/scala/Dispatcher.scala

```diff
@@ -0,0 +1,41 @@
+import scala.io.Source
+
+import telegram.API._
+import tinkoff.API._
+
+/**
+  * Main polling loop.
+  */
+object Dispatcher extends App {
+
+  def getFormattedRates(rates: List[Rate]): String = {
+    val textRatesList = rates.map(x => "%3s: %7.3f  %7.3f".format(x.fromCurrency.name, x.buy, x.sell.get))
+    "<pre>" +
```

Comment:
There should not be a newline after \<pre\>, so only way to do it and not break indents is:
```
    s"""<pre>       Buy      Sell
            |      ------   ------
            |${textRatesList.mkString("\n")}
            |</pre>""".stripMargin
```
Is it better?